script/add-left-nav-collections.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-continue, no-param-reassign */
+
+/**
+ * Writes a human-readable JavaScript file containing build properties available globally under `window.settings`.
+ * @param {object} options The build options as passed to the build script and processed through Metalsmith.
+ */
+function addLeftNavCollections() {
+  return (files, metalsmith, done) => {
+    Object.keys(files).forEach(key => {
+      const file = files[key];
+      const splitPath = key.split('/');
+
+      if (splitPath.length === 5) {
+        splitPath.pop();
+
+        const parentFile = `${splitPath.join('/')}.md`;
+
+        Object.assign(file, {
```

Comment:
@jbalboni It's the same thing but I programed it this way initially if I needed to modify more then 1 property but since it's only 1 property I think it's appropriate to use your suggestion.

Reply:
It may not be necessary here, but if you find yourself messing with file paths often, you can use Node's `path` module instead of strings. It can be friendlier to work with and make your code more meaningful. For example, to get the parent directory, you can do this -

```js
const fileName = key;
const currentDir = path.dirname(fileName);
const parentDir = path.join(currentDir, '../');
const parentFile = `${parentDir}.md`
```

Do you also have to check of index files? I don't think we do that once we're five-levels deep, but not sure.

Reply:
@ncksllvn Totally forgot I can use that.